src/nrrd/nrrd.h

```diff
@@ -429,6 +430,10 @@ typedef struct NrrdIoState_t {
   void *oldData;            /* ON READ: if non-NULL, pointer to space that
                                has already been allocated for oldDataSize */
   size_t oldDataSize;       /* ON READ: size of mem pointed to by oldData */
+  size_t  numberOfElementsPerChunk;    /* if != 0, a subset will be read or written and it indicates the
```

Comment:
As we always read/write only one single chunk at a time and we can change chunk size between reads/writes, the name "per chunk" may be confusing (it would suggest that there are multiple chunks).

Instead, chunkSize or chunkElementCount would be probably more clear.

Reply:
ok